packages/strapi-plugin-content-manager/admin/src/components/SelectWrapper/index.js

```diff
@@ -206,6 +231,12 @@ function SelectWrapper({
 
   const to = `/plugins/${pluginId}/collectionType/${targetModel}/${value ? value.id : null}`;
 
+  const searchToPersist = useMemo(() => {
```

Comment:
Not necessary.

Pous query is an object and paramsToKeep is also an object. Their refs might change